mne/epochs.py

```diff
@@ -1362,18 +1362,22 @@ def tmax(self):
 
     def __repr__(self):
         """Build string representation."""
-        s = 'n_events : %s ' % len(self.events)
+        s = ' %s events ' % len(self.events)
```

Comment:
Wouldn't it be more pythonic to give the individual elements proper names (n_chans, t_lims, ...) and use `join`?

Reply:
I think the goal should be to optimize for compactness and readability.

In addition to not necessarily increasing readability (and decreasing compatness), using variable-like such as `n_chans` can improperly imply that such variables exist as attributes of the object, too.

Reply:
ok